orchestration/src/main/java/com/sap/ai/sdk/orchestration/UserMessage.java

```diff
@@ -13,5 +18,90 @@ public class UserMessage implements Message {
   @Nonnull String role = "user";
 
   /** The content of the message. */
-  @Nonnull String content;
+  @Nonnull
+  @Getter(onMethod_ = @Beta)
+  MessageContent content;
+
+  /**
+   * Creates a new user message from one or more strings.
+   *
+   * @param message the first message.
+   * @param additionalMessages the additional messages.
+   */
+  public UserMessage(@Nonnull final String message, @Nullable final String... additionalMessages) {
+    content = MessageContent.text(message, additionalMessages);
+  }
+
+  /**
+   * Creates a new user message with the given message content.
+   *
+   * @param messageContent the message content.
+   */
+  public UserMessage(@Nonnull final MessageContent messageContent) {
```

Comment:
(Minor)

Please omit default constructors that are generated by lombok.

Since, lombok notices a custom constructor above, you would need to mark it with `@Tolerate`, otherwise we'll not have the default constructor via `@Value`